index.html

```diff
@@ -5,118 +5,122 @@
+
+    <link href="./normalize.css" rel="stylesheet" type="text/css">
     <link href="./rapido.css" rel="stylesheet" type="text/css">
   </head>
-  <body class="rapido">
+  <body>
     <main>
-      <header>
-        <button><a href="#">☰</a></button>
-        <nav class="nav">
-          <ul>
-            <li>
-              <a href="https://github.com/nextbitlabs/Rapido">
-                View on Github
-                <img src="./media/github.svg">
-              </a>
-            </li>
-          </ul>
-        </nav>
-        <button><a href="#">close menu</a></button>
-        <img src="./media/icon.svg"></img>
-      </header>
-      <article>
+      <article class="rapido">
         <header>
-          <h1>Welcome to Rapido</h1>
-          <p><strong>Rapido</strong> helps you to write web documents with the only use of HTML snippets, it is designed with a focus on usability and frugality.</p>
-          <p>The present documentation has been created following Rapido guidelines and nothing more, you can easily get your own documents with the same layout and style. Rapido guidelines are described in the next sections.</p>
+          <h1>Rapido: semantic HTML elements for your writings.</h1>
+          <p>Rapido provides a default style for a set of <strong>semantic HTML</strong> elements that you can use to compose your essays. No CSS or JS is required.</p>
+          <p>The present document has been created using Rapido, you can easily get documents or blog posts with the same layout and style. See how to use Rapido in the next sections.</p>
+          <p>Rapido is open source, view it on <a href="https://github.com/nextbitlabs/Rapido">Github</a>.</p>
         </header>
+        <section>
+          <h1>Motivation</h1>
+          <p>Having your own corner on the web where you can freely share thoughts or technical stuff <a href="https://medium.com/@dan_abramov/why-my-new-blog-isnt-on-medium-3b280282fbae">is still cool</a>, and the popularity of tools like <a href="https://jekyllrb.com/">Jekyll</a> or <a href="https://www.gatsbyjs.org/">Gatsby</a> is proof of that. Rapido intervenes there, offering semantic HTML templates for a variety of typical author needs: like header, sections, side notes, references and so on.<small>The importance of using <a href="https://developer.mozilla.org/en-US/docs/Glossary/semantics#Semantics_in_HTML">semantic HTML</a> is a common theme in web development, it is a good idea to use the relevant HTML element for the job. Semantic HTML improves the usability of the code and ensures maximum <a href="https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML">accessibility</a>.</small></p>
```

Comment:
I think that "proves that" might be better.

Reply:
Done.